coin/coins.go

```diff
@@ -1,13 +1,18 @@
 // Code generated by go generate; DO NOT EDIT.
 // This file was generated by robots at
-// 2021-06-18 08:30:36.901143 -0700 PDT m=&#43;0.001565402
+// 2021-08-17 09:18:06.774336 &#43;0300 EEST m=&#43;0.002442799
 // using data from coins.yml
 package coin
 
 import (
 	"fmt"
 )
 
+const (
```

Comment:
can we move prefix into `asset` lib?

Reply:
You mean asset package? 
There are prefixes already, but I had to duplicate them here to avoid circular dependencies.

Reply:
oh I see, that make sense then